app/Models/Asset.php

```diff
@@ -1068,8 +1068,14 @@ public function scopeOverdueForAudit($query)
 
     public function scopeDueOrOverdueForAudit($query, $settings)
     {
+        if(is_null($settings->audit_warning_days)){
+            $days_left = 0;
```

Comment:
This variable name is a little confusing, since it's not really days left, which implies some math has been done on it to determine some magical number. Maybe just call it `$interval`? (I know it's a small thing, but I'll forget and it will confuse me in a year. :D)

Reply:
Could also maybe do something like:

```
$interval = 0 + $settings->audit_warning_days;
```

tho not sure if it would choke on the nullness.

Reply:
I could try, I like it.

Reply:
Or actually a better solution would probably be:

`$interval = $settings->audit_warning_days ?? 0;`